code/datums/components/milkable.dm

```diff
@@ -0,0 +1,67 @@
+// how cows and other simplemobs are able to be milked
+
+/datum/component/milkable
+	var/mob/producer
+	var/obj/item/reagent_containers/intstorage
+	var/max_reagents
+	var/min_reagents
+	var/datum/reagent/reagent_generated
+	var/datum/callback/on_milked
```

Comment:
This callback is never used
If the use of it is ever needed you can just readd it

Reply:
Same goes to containers